modules/govuk_ci/manifests/agent.pp

```diff
@@ -54,4 +54,10 @@
   package { 'libgdal-dev': # needed for mapit
     ensure => installed,
   }
+
+  cron::crondotdee { 'clean_up_bundles':
+    command => 'find /var/lib/jenkins/bundles/* -maxdepth 0 -type d -mtime +1 -exec rm -rf {} \;',
```

Comment:
Ooh, it's probably because with sudo wildcard expressions can't be expanded. As root it works as you expected.